### PR TITLE
navbar setup with login functionality

### PR DESCRIPTION
**Pull request by Talhaaa99**

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+  const [password, setPassword] = useState('');
+  const [mfaCode, setMfaCode] = useState('');
+  const [error, setError] = useState('');
+  const [needsMfa, setNeedsMfa] = useState(false);
+  const [preAuthToken, setPreAuthToken] = useState('');
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [userInitials, setUserInitials] = useState('');
+
+  useEffect(() => {
+    const storedEmail = localStorage.getItem('email');
```
Comment (prabhanukash) [inline src/components/Navbar.tsx:32]: instead if email do check with token

File: src/services/werate-api.ts [hunk 2]
```diff
@@ -1,39 +1,151 @@
...
+    const response = await axios.post(
+      `${API_BASE_URL}/v1/register`,
+      { email, password },
+      { headers: { ['Content-Type']: ['application/json'] } }
+    );
+    return response.data;
+  } catch (error) {
+    console.error('Error during registration:', error);
+  }
+};
```
Comment (prabhanukash) [inline src/services/werate-api.ts:110]: move this auth service

**Review by prabhanukash — CHANGES_REQUESTED** [review]
please address comments

File: src/services/werate-api.ts (hunk 2 above)
Comment (Talhaaa99) [reply src/services/werate-api.ts]: moved this

File: src/components/Navbar.tsx (hunk 1 above)
Comment (Talhaaa99) [reply src/components/Navbar.tsx:32]: using this to get initials, to avoid 2 api calls for faster response time

**Comment by verschaevesiebe** [comment]
Please resolve conflicts @prabhanukash or @Talhaaa99

**Comment by verschaevesiebe** [comment]
Looks good ! :)

**Pull request merged**